cenpy/legacy_tiger.py

```diff
@@ -0,0 +1,229 @@
+import geopandas as gpd
+from warnings import warn
+from gdal import osr, ogr
+import shutil
+
+from urllib.request import urlopen
+from zipfile import ZipFile
+import os
+import tempfile
+
+
+class FTPConnection(object):
+
+    def __init__(self, year, statenum, stateabbr, county):
+
+        url = {2000: 'ftp://ftp2.census.gov/geo/tiger/TIGER2003/{statenum}_{stateabbr}/tgr{statenum}{county}.zip' .format(statenum=statenum, stateabbr=stateabbr, county=county),
+               1990: 'ftp://ftp2.census.gov/geo/tiger/TIGER1992/{statenum}/{statenum}{county}.zip' .format(statenum=statenum, stateabbr=stateabbr, county=county)
+               }
+
+        self.url = url[year]
+        self.statenum = statenum
+        self.stateabbr = stateabbr
+        self.county = county
+
+    def _tiger_to_tract(self, infile):
+
+        # Modified from original at
+        # https://svn.osgeo.org/gdal/tags/1.4.3/gdal/pymod/samples/tigerpoly.py
+
+        class Module:
+            def __init__(self):
```

Comment:
Dumb superstition, but can you make the `object` explicit in class init and call `self` something else, like `mod`?
```python
class Module(object):
    def __init__(mod):
        mod.lines = {}
        mod.poly_line_links = {}
```
I just don't like when there are two variables in nested scopes, but named the same thing. If ever this had to be debugged, I wouldn't be able to easily tell which `self` an error message refers to. I try to avoid this when I can.